crates/utils/src/error.rs

```diff
@@ -129,7 +129,8 @@ pub enum LemmyErrorType {
   InvalidUrlScheme,
   CouldntSendWebmention,
   ContradictingFilters,
-  InstanceBlockAlreadyExists,
+  InstanceBlockCommunitiesAlreadyExists,
+  InstanceBlockPersonsAlreadyExists,
```

Comment:
Does this really need a separate error message? Imo it would be enough with a single error `AlreadyExists` which is used in all cases and replaces stuff like `CommunityModeratorAlreadyExists`, `PersonBlockAlreadyExists` etc. Similar to the single `NotFound` error. In the same way there could be a single error `CouldntCreateItem` and `CouldntUpdateItem`.